src/components/guidance-content/help/HelpGeneral.js

```diff
@@ -0,0 +1,724 @@
+                periods; there are six such periods from 1960 to 2100.
+                This data is available for all of Canada.
+              </p>
+              <h5>Variable names and meanings</h5>
+              <ListGroup>
+                <ListGroupItem header='pr'>
+                  Precipitation at ground level
+                </ListGroupItem>
+                <ListGroupItem header='tasmax'>
+                  Daily maximum near-surface air temperature
+                </ListGroupItem>
+                <ListGroupItem header='tasmin'>
+                  Daily minimum near-surface air temperature
+                </ListGroupItem>
+              </ListGroup>
+            </ListGroupItem>
+
+            <ListGroupItem header='Climdex (climate extremes indices)'>
+              <p>
+              Measures of weather extremes calculated from model output.
+              Climdex defines 27 <a href='https://www.climdex.org/indices.html'>climate extremes indices</a>,
+              encompassing extreme precipitation, extremes of temperature,
+              or lack thereof in different ways.
+              This is a heterogeneous dataset; a given extreme may be calculated
+              either as a monthly index averaged over thirty year periods
+              from 1960 to 2100, or an annual index calculated individually
+              for each year from 1950 to 2100.
+              This data is available for all of Canada.
+              </p>
+              <h5>Variable names and meanings</h5>
+              <ListGroup>
+                <ListGroupItem header='altcddETCCDI'>
+                  <p>
+                    Maximum number of consecutive days per year with less than 1mm
+                    of precipitation
+                  </p>
+                  <p>
+                    This is a variant ('alt') version of the <a href='https://www.climdex.org/indices.html'>Climdex</a> variable <em>CDD</em>.
+                  </p>
+                </ListGroupItem>
+                <ListGroupItem header='altcsdiETCCDI'>
+                  <p>
+                    Cold spell duration index spanning years
+                  </p>
+                  <p>
+                    This is a variant ('alt') version of the <a href='https://www.climdex.org/indices.html'>Climdex</a> variable <em>CSDI</em>.
+                  </p>
+                </ListGroupItem>
+                <ListGroupItem header='altcwdETCCDI'>
+                  <p>
+                    Maximum number of consecutive days per year with at least 1mm of
+                    precipitation
+                  </p>
+                  <p>
+                    This is a variant ('alt') version of the <a href='https://www.climdex.org/indices.html'>Climdex</a> variable <em>CDD</em>.
+                  </p>
+                </ListGroupItem>
+                <ListGroupItem header='altwsdiETCCDI'>
+                  <p>
+                    Warm spell duration index spanning years
```

Comment:
Why do we even have this variable? There isn't gonna be a warm spell from December to January, is there?
Anyway.

"This is an alternative version of the Climdex variable csdi that counts the full length of cold spells that extend from December of the target yeaar through January of the next year."

Reply:
Not in the northern hemisphere ... at least not until global warming gets much worse.